libsplinter/src/biome/rest_api/config.rs

```diff
@@ -29,7 +29,7 @@ pub struct BiomeRestConfig {
     /// Duration of JWT tokens issued by this service
     access_token_duration: Duration,
     #[cfg(feature = "biome-credentials")]
-    /// Cost for encripting users password
+    /// Cost for encrypting user's password
```

Comment:
Typo in commit message (also the summary is very long).

Reply:
The "typo" in the commit message was the typo being fixed in the commit. I've updated the message to make that clearer